travis/licenses_golden/licenses_flutter

```diff
@@ -1062,6 +1062,8 @@ ORIGIN: ../../../LICENSE
 TYPE: LicenseType.bsd
 FILE: ../../../flutter/.travis.yml
 FILE: ../../../flutter/DEPS
+FILE: ../../../flutter/frontend_server/bin/starter.dart
+FILE: ../../../flutter/frontend_server/lib/server.dart
```

Comment:
Those end up in the binaries that are used for development only.

If I take those back out, license check complains: https://travis-ci.org/flutter/engine/builds/265742542?utm_source=github_status&utm_medium=notification